auto_update_scheduled.yaml

```
@@ -489,888 +630,866 @@
+            | rejectattr('entity_id', 'in', os_update_entity)
+            | rejectattr('entity_id', 'in', supervisor_update_entity)
+            | rejectattr('entity_id', 'in', firmware_update_entities)
+            | rejectattr('entity_id', 'in', input_update_exclusions)
+            | map(attribute='entity_id')
+            | list
+            | first
+          }}
+        {% endif %}
+      combined_list: >-
+        {% set entities = namespace(list=[]) %}
+        {%- for entity in [firmware_update_entities, general_update_entities, 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)